uwblueprint/rowan-house
# Fix: button font styling

## Implementation Description
- Change the font styling in the `button.ts` theme customization

## Screenshots
<img width="929" alt="Screen Shot 2022-03-11 at 9 15 32 PM" src="https://user-images.githubusercontent.com/40974372/158004991-db55dd1a-07d0-467a-aad5-9847bd89321b.png">


## What should reviewers focus on?
- Check that this aligns with the Figma file -> https://www.figma.com/file/zh6zk2LJo0RodIHBVycylC/RHS-Design-System?node-id=210%3A13900
  - Was padding right? It's not consistent between the Figma and here, but pretty consistent within the file
- Check that this is the proper way of styling components

## Testing Procedure
- What test(s) should we run to make sure your code works?
   - What ways could the input be wrong?
   - Is it secure? Could nefarious users access/break it?

Can add into any file and check that the appearances look write
```ts
import { Button } from "@chakra-ui/react";

<Button variant="default">Default</Button>
<Button variant="md">md</Button>
<Button variant="sm">sm</Button>
<Button variant="xs">xs</Button>
<Button variant="disabled">disabled</Button>
<Button variant="outline-lg">outline lg</Button>
<Button variant="outline-md">outline md</Button>
<Button variant="outline-sm">outline sm</Button>
<Button variant="outline-xs">outline xs</Button>
<Button variant="outline-disabled">outline disabled</Button>
```

## Things to Note
- Additional dependencies/libraries you've added?
- Things you'd want to know when coming back to the code after a few months

None

## Checklist
